bin/martinize2

```diff
@@ -628,9 +628,9 @@ def entry():
     prot_group.add_argument(
         "-scfix",
         dest="scfix",
-        action="store_true",
+        action="store_false",
         default=True,
-        help="Apply side chain corrections.",
+        help="[don't] apply side chain corrections.",
     )
```

Comment:
So, the name of the flag remains the same, but the behaviour is inverted? I'm not sure that's very insightful

Reply:
change name to `-noscfix`?

Reply:
That would work for me.